Cargo.toml

```diff
@@ -23,30 +23,30 @@ aurora-engine-sdk = { git = "https://github.com/aurora-is-near/aurora-engine.git
 base64 = "0.13.0"
 borsh = { version = "0.9.3" }
 byteorder = "1.4.3"
-clap = { version = "3.2.7", features = ["derive"] }
-derive_builder = "0.10.2"
+clap = { version = "4", features = ["derive"] }
+derive_builder = "0.12.0"
 engine-standalone-storage = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.8.0", default-features = false }
-fixed-hash = "0.7.0"
+fixed-hash = "0.8.0"
 futures = "0.3.5"
 hex = "0.4.3"
-impl-serde = "0.3.2"
+impl-serde = "0.4.0"
 itertools = "0.10.3"
 keccak-hasher = "0.15.3"
 lazy_static = "1.4.0"
-lru = "0.7.3"
+lru = "0.8.1"
 near-crypto = { git = "https://github.com/near/nearcore", tag = "1.26.1" }
 near-indexer = { git = "https://github.com/near/nearcore", rev = "313693c442c3081537b51ebc9044800d590aea6b" }
 near-primitives = { git = "https://github.com/near/nearcore", tag = "1.26.1" }
-near-lake-framework = "0.3.0"
-primitive-types = "0.7.2"
+near-lake-framework = "0.5.2"
```

Comment:
Updating this makes me a little nervous because we don't have any good way to test the Near Data Lake feature as far as I know.